tester/lib/cli.js

```diff
@@ -65,7 +65,7 @@ export async function main(argv = process.argv.slice(2)) {
   };
   for (const globSet of globs) {
     fileSets.push(
-      await glob(globSet, globOptions)
+        await glob(globSet, globOptions)
```

Comment:
looks strange, if tab size is 2 spaces we should keep it as is.

Reply:
I'm not sure I'm using the right formatter with theproper config...